src/main/java/sjchat/restapi/entities/Message.java

```diff
@@ -0,0 +1,40 @@
+package sjchat.restapi.entities;
+
+public class Message {
+  private long id;
+  private String message;
+  private long user;
+
+  public Message() {
+  }
+
+  public Message(long id, String message, long user) {
+    this.id = id;
+    this.message = message;
+    this.user = user;
+  }
+
+  public long getId() {
+    return id;
+  }
+
+  public void setId(long id) {
```

Comment:
Consider using the [builder pattern](https://en.wikipedia.org/wiki/Builder_pattern#Java) instead of setters. Immutability is a nice thing.